src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kalecgos.cpp

```diff
@@ -562,8 +561,7 @@ class kalecgos_teleporter : public GameObjectScript
 
         if (player->HasAura(AURA_SPECTRAL_EXHAUSTION) || SpectralPlayers >= MAX_PLAYERS_IN_SPECTRAL_REALM)
         {
-            player->GetSession()->SendNotification(GO_FAILED);
-            return true;
+            return SPELL_FAILED_CANT_DO_THAT_RIGHT_NOW;
```

Comment:
Thanks. I removed the message altogether. Let's see if the result is acceptable.